runtime/westend/src/lib.rs

```diff
@@ -328,6 +328,20 @@ impl pallet_session::historical::Config for Runtime {
 	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
 }
 
+pub struct MaybeSignedPhase;
+
+impl Get<u32> for MaybeSignedPhase {
+	fn get() -> u32 {
+		// 1 day = 4 eras -> 1 week = 28 eras. We want to disable signed phase once a week to test the fallback unsigned
+		// phase is able to compute elections on Westend.
+		if Staking::current_era().unwrap_or(1) as u32 % 28 == 0 {
```

Comment:
```suggestion
		if Staking::current_era().unwrap_or(1) % 28 == 0 {
```

`EraIndex` is already u32, no?